h5grove/flaskutils.py

```diff
@@ -57,15 +57,14 @@ def attr_route():
     """`/attr/` endpoint handler"""
     filename = get_filename(request)
     path = request.args.get("path")
-    format_arg = request.args.get("format")
```

Comment:
Removing `format`query param from all endpoints not supposed to have it....

Reply:
Good point !